website/i18n/en.json

```diff
@@ -45,6 +49,10 @@
         "title": "Database Connection",
         "sidebar_label": "Database Connection"
       },
+      "deploy-from-github": {
```

Comment:
This should be a subpage of CI/CD.

Reply:
This is my first time using Docusourous. I have not quite figured out how to get the nested navigation working despite reading over the official docs. I may need your assistance on this.

Reply:
Okay, figured it out. Version 1.x of Docusourus is being used instead of 2.x.

Reply:
I've moved renamed, re-titled and moved the existing CI/CD Integration page as being a nested page under CI/CD Integration subcategory page and now referring to it as Deploy from GitLab as this page almost completely is devoted to GitLab, with the exception of the "Alternative Method" at the bottom of the page. This visually looks better to me in the navigation as we now have both Deploy from GitHub as well as a Deploy from GitLab page. I'll be working on an automation tool for both GitLab and Bitbucket in the very near future, to which we can then update the existing GitLab page for simplicity for users and add support for Bitbucket. For now, hope this is more in line with what you invision. Let me know.